core/payment/Cargo.toml

```diff
@@ -10,7 +10,7 @@ gnt-driver = []
 dummy-driver = []
 
 [dependencies]
-ya-core-model = { version = "0.1", features = ["payment", "market", "identity", "with-diesel"] }
+ya-core-model = { version = "0.1", features = ["payment", "market", "identity"] }
 ya-client-model = "0.1"
```

Comment:
don't you need?
```suggestion
ya-client-model = { version = "0.1", features = [ "with-diesel" ] }
```

Reply:
Looks like it's needed, but the compiler doesn't complain... :thinking:

Reply:
try to compile from within the crate directory... ;)

Reply:
Oh, right. I think Cargo should detect this nevertheless. If building of any of the crates alone fails, so should building the whole workspace.